docs/dev/gcp.md

```diff
@@ -5,21 +5,21 @@
 - Install gcloud CLI: [Instructions](https://cloud.google.com/sdk/docs/install)
 - Authentication:
 
-    Packer plugin for google compute requires authenticate with GCP using service account credentails. Instructions to create service account and credentials file can be found [here](https://www.packer.io/plugins/builders/googlecompute#running-outside-of-google-cloud)
+    Packer plugin for Google Compute requires authenticate with GCP using service account credentials. Instructions to create service account and credentials file can be found [here](https://www.packer.io/plugins/builders/googlecompute#running-outside-of-google-cloud)
 
     **Using CLI:**
     ```bash
     export USER=<SERVICE_ACCOUNT_USER>
     export GCP_PROJECT=<GCP_PROJECT_NAME>
-    export GOOGLE_APPLICATION_CREDENTIALS=$HOME/.gcp/credentials.json
+    export GOOGLE_APPLICATION_CREDENTIALS=$HOME/.gcloud/credentials.json
```

Comment:
we mount `$HOME/.gcp` directory on devkit container when creating GCP images. https://github.com/mesosphere/konvoy-image-builder/blob/main/Makefile#L65

if we want to use `.gcloud`, we will need to fix the Makefile.

Reply:
Seems like a reasonable change, as `$HOME/.gcloud` is the canonical location for credentials. I added a commit.

This change does not affect CI, which gets credentials from a [different location](https://github.com/mesosphere/konvoy-image-builder/blob/00df6baa09335e0778e3b3a828fb6fec4f1bb675/Makefile#L68-L69).